tests/App/Entity/GalleryHasMedia.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\MediaBundle\Tests\App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Sonata\MediaBundle\Entity\BaseGalleryHasMedia;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="media__gallery_has_media")
+ */
+class GalleryHasMedia extends BaseGalleryHasMedia
```

Comment:
GalleryItem. This class needs to be renamed, and fixed for all the gallery_had_media mentions